patches/chrome-browser-ui-views-tabs-tab_hover_card_bubble_view.cc.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/chrome/browser/ui/views/tabs/tab_hover_card_bubble_view.cc b/chrome/browser/ui/views/tabs/tab_hover_card_bubble_view.cc
+index 4c9bcb9373983..0276340fae5c2 100644
+--- a/chrome/browser/ui/views/tabs/tab_hover_card_bubble_view.cc
++++ b/chrome/browser/ui/views/tabs/tab_hover_card_bubble_view.cc
+@@ -775,7 +775,8 @@ TabHoverCardBubbleView::TabHoverCardBubbleView(Tab* tab)
+   domain_label_ = AddChildView(
+       std::make_unique<FadeLabel>(views::style::CONTEXT_DIALOG_BODY_TEXT, 1));
+ 
+-  if (TabHoverCardController::AreHoverCardImagesEnabled()) {
++  if (TabHoverCardController::AreHoverCardImagesEnabled()
```

Comment:
`#define AreHoverCardImagesEnabled AreHoverCardImagesEnabled() || brave_tabs::AreCardPreviewsEnabled( \
         tab->controller()->GetBrowser()->profile()->GetPrefs`

Reply:
does this still work if `kTabHoverCardImages` feature is disabled? If not this needs to be `&&` instead of `||`

Reply:
It does, yeah, and it's intended to :smile:

Reply:
I'm still not sure the conditional is right. If the feature is enabled then won't it ignore our setting and always use hover card?